packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -1,11 +1,11 @@
-import { useEffect } from 'react'
+import { useEffect, useState } from 'react'
 
 import { accountSelectors, Status } from '@audius/common'
+import { getHasAccount } from '@audius/common/dist/store/account/selectors'
```

Comment:
I think this import is wrong, we should be destructuring from accountSelectors or something like that

Reply:
My bad thx